src/ServicePulse.Host/vue/src/stores/FailedMessageStore.js

```diff
@@ -13,8 +13,8 @@ export const useFailedMessageStore = defineStore("FailedMessageStore", {
     },
   },
   getters: {
-    serviceControlId: (state) => state.failedMessagesList[0].id,
-    errorCount: (state) => state.failedMessagesList[0].count,
-    isFailedMessagesEmpty: (state) => state.errorCount === 0,
+    serviceControlId: (state) => state.failedMessagesList[0]?.id,
```

Comment:
I didn't investigate what was going on here, but this was throwing tonnes of errors into my console, so I made it capable of handling null values. There is something very wrong with the way this state is set/handled, since the UI flickers constantly.

Reply:
We fixed the flicker. Is this the same thing that was being fixed in https://github.com/Particular/ServicePulse/pull/1691 ?